web-local/package.json

```diff
@@ -29,6 +29,8 @@
     "@babel/preset-env": "^7.16.11",
     "@playwright/test": "^1.28.1",
     "@rollup/plugin-typescript": "^8.3.1",
+    "@stdlib/random-base": "^0.0.6",
+    "@stdlib/random-shuffle": "^0.0.7",
```

Comment:
@hamilton I changed this from all of `stdlib` to just the two submodules that are actually being used in the dev route, which is much much slimmer. I can remove it if you want, but it's very useful to be able to draw samples from a seedable RNG for this kind of prototyping, and I don't think including adding just these submodules as dev dependencies has any substantial downsides at this point-- it adds a couple seconds perhaps to `npm install`, but should not impact end user experience. But let me know me know if you want it out, I can come up with a workaround / roll my own seedable RNG that's good enough for this application.